linera-execution/src/runtime.rs

```diff
@@ -1601,6 +1611,13 @@ impl ServiceSyncRuntime {
         context: QueryContext,
         deadline: Option<Instant>,
     ) -> Self {
+        // Query the allow_contract_logs setting from the execution state.
+        let allow_contract_logs = execution_state_sender
+            .send_request(|callback| ExecutionRequest::AllowContractLogs { callback })
+            .ok()
+            .and_then(|receiver| receiver.recv_response().ok())
+            .unwrap_or(false);
```

Comment:
(Can't it be passed into `new_with_deadline`, too? When I suggested a field in the runtime I was hoping to get rid of the execution state actor request altogether. If we can't get rid of that, maybe it's cleaner to always use it and not have the boolean field at all?)